src/pymor/analyticalproblems/expressions.py

```diff
@@ -273,6 +314,31 @@ def numpy_expr(self):
 
         return f'({self.first.numpy_expr()}{first_ind} {self.numpy_symbol} {self.second.numpy_expr()}{second_ind})'
 
+    def fenics_expr(self, params):
+        if isinstance(self.fenics_op, str):
```

Comment:
Both if-branches essentially contain the same code. So you should only need one branch where you write something like

```python
ufl_op = getattr(ufl, self.fenics_op) if isinstance(self.fenics_op, str) else self.fenics_op
```